projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -106,6 +107,30 @@ const RecordMeterBar: React.FC< RecordMeterBarProps > = ( {
 					} ) }
 				</ul>
 			</div>
+			<table
+				className="screen-reader-text"
+				summary="This is the screen reader alternative data table for the record meter bar graphic."
```

Comment:
I think it may be best to omit the summary. From https://bcourses.berkeley.edu/courses/1476601/pages/tables:

> This attribute is not needed for a regularly formatted table

Reply:
Yep makes sense! Pulling it out shortly.